src/main/java/com/revealprecision/revealserver/api/v1/controller/ReportDashboardController.java

```diff
@@ -40,11 +40,12 @@ public List<String> getReportTypes(@PathVariable("identifier") UUID identifier)
     return ApplicableReportsEnum.valueOf(plan.getInterventionType().getCode()).getReportName();
   }
 
+
   @GetMapping("/reportData")
   public ResponseEntity<FeatureSetResponse> getDataForReports(
       @RequestParam(name = "reportType") String reportType,
       @RequestParam(name = "planIdentifier") UUID planIdentifier,
-      @RequestParam(name = "parentIdentifier", required = false) UUID parentIdentifier,
+      @RequestParam(name = "parentIdentifier", required = false) String parentIdentifier,
```

Comment:
why are moving from uuid to string?

Reply:
here when traversing through the hierarchy of the dashboard....country, province etc....we eventually get to supervisor and cdd....these "parents" are no longer UUID's....their identifiers are now string